src/main/java/com/google/sps/servlets/EnterQueue.java

```diff
@@ -50,7 +62,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
       String idToken = request.getParameter("idToken");
       FirebaseToken decodedToken = authInstance.verifyIdToken(idToken);
-      String uID = decodedToken.getUid();
+      String userID = decodedToken.getUid();
 
       if (request.getParameter("enterTA") == null) {
         int retries = 10;
```

Comment:
Is this ever used?

Reply:
Used to check if the TA's ID is stored in the list of TA of the class.